src/scry/completion/dependency_graph.cr

```diff
@@ -25,13 +25,25 @@ module Scry::Completion::DependencyGraph
   end
 
   class Graph
+    getter prelude_node : Node
+
     def initialize(@nodes = {} of String => Node)
+      if crystal_path = ENV["CRYSTAL_PATH"]?.to_s.split(":").last?
+        prelude_path = "#{crystal_path}/prelude.cr"
```

Comment:
Why not use the compiler's `Crystal::CrystalPath` class to find files?

Reply:
@bew Thank you! I think you're right, let me fix that :+1:

Reply:
hopefully it doesn't depends on the whole compiler ^^

Reply:
Done! :tada: 

However, I guess `ENV["CRYSTAL_PATH"]` is still [required to trace workspace libs](https://github.com/crystal-lang-tools/scry/blob/master/src/scry/initializer.cr#L14)

Reply:
won't Crystal::CrystalPath give the path of crystal when you're compiling and not in the context of where scry is running?

Reply:
Hi @laginha87, I'm using [`Crystal::CRYSTAL_PATH`](https://github.com/crystal-lang/crystal/blob/1f3e8b0e742b55c1feb5584dc932e87034365f48/src/compiler/crystal/command/env.cr#L11), and this constant still uses `ENV["CRYSTAL_PATH"]` under the hood, because even [`Crystal::CrystalPath`](https://github.com/crystal-lang/crystal/blob/master/src/compiler/crystal/crystal_path.cr#L9) uses it, and also uses [`Crystal::Config.path`](https://github.com/crystal-lang/crystal/blob/master/src/compiler/crystal/crystal_path.cr#L9) that catches `CRYSTAL_CONFIG_PATH` at compile time, but don't worry about that, because `ENV["CRYSTAL_PATH"]` is checked first an we always set this env variable :sweat_smile: